src/test/CMakeLists.txt

```diff
@@ -302,13 +302,18 @@ include_directories(../../third_party/googletest/googletest/)
 
 include_directories(${CMAKE_CURRENT_SOURCE_DIR})
 
+add_custom_target(resourceLink
```

Comment:
Doc with `# For convenience, to be able to launch, e.g., hyriseTest, directly from the build directory, add a link to the resources/ directory at the root of the build directory)

Reply:
Good point.